cardano-db-sync/src/Cardano/DbSync/Types.hs

```diff
@@ -25,6 +25,8 @@ module Cardano.DbSync.Types
   , ShelleyTxOut
   , ShelleyTxSeq
   , SocketPath (..)
+  -- * Re-exports (we don't need 15 deps)
+  , Tip
```

Comment:
Re-exports of types from other packages are *always* a bad.

Reply:
Removed.